{"name": "test_login_2_wrong[data1]", "status": "broken", "statusDetails": {"message": "AttributeError: 'NoneType' object has no attribute 'clear'", "trace": "self = <test_login.TestLogin object at 0x035062B0>\ndata = {'expected': '请输入正确的手机号', 'password': '', 'phone': 123}\nmy_setup_class = (<selenium.webdriver.chrome.webdriver.WebDriver (session=\"716e5ccc75003eb7595dc6767d0217d0\")>, <pages.login.LoginPage object at 0x034B6270>)\n\n    @pytest.mark.usefixtures('my_setup_class')\n    # @pytest.mark.usefixtures('init_driver')\n    @pytest.mark.login\n    @pytest.mark.parametrize('data',login.user_incorrent)\n    # @data(*login.user_incorrent)\n    def test_login_2_wrong(self,data,my_setup_class):\n        driver,login_page=my_setup_class\n>       login_page.clear_phone()\n\ntest_login.py:38: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pages.login.LoginPage object at 0x034B6270>\n\n    def clear_phone(self):\n>       return self.get_phone_element().clear()\nE       AttributeError: 'NoneType' object has no attribute 'clear'\n\npages\\login.py:26: AttributeError"}, "attachments": [{"name": "log", "source": "713181f7-9724-4e32-9650-ff0b80f6fc39-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'phone': 123, 'password': '', 'expected': '请输入正确的手机号'}"}], "start": 1610847447372, "stop": 1610847468050, "uuid": "cf519150-02b7-4bca-afb0-4618170f6b3c", "historyId": "b80ecc7563f66e2063562f0a48b43028", "testCaseId": "ffcf0d5ae779a52948f63f6eb8905898", "fullName": "test_login.TestLogin#test_login_2_wrong", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('my_setup_class')"}, {"name": "tag", "value": "login"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "PC201809011422"}, {"name": "thread", "value": "1180-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}